test/mock_clj/core_test.clj

```diff
@@ -43,7 +43,8 @@
   (with-mock [foo 5]
     (foo 1 2 3)
     (foo 5 6 6)
-    (is (calls foo) [[1 2 3] [5 6 6]])))
+    (is (= (calls foo) [[1 2 3] [5 6 6]]))
+    (is (not (= (calls foo) [[1 2 3] [5 2 6]])))))
```

Comment:
Thank you for the fix! 

It seems the assertion on line 46 already suffice in this case.

Reply:
I shall update it.